src/app/teams/teams-view.component.ts

```diff
@@ -215,10 +215,13 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
   }
 
   changeMembership(type, memberDoc?) {
-    this.changeMembershipRequest(type, memberDoc).subscribe((message) => {
-      this.setStatus(this.team, this.userService.get());
-      this.planetMessageService.showMessage(message);
-    });
+    return forkJoin([
+      this.changeMembershipRequest(type, memberDoc).subscribe((message) => {
```

Comment:
In a `forkJoin` there should not be a `.subscribe` for the observables.  The subscription comes later